source/_meta/_head.hbs

```diff
@@ -9,7 +9,17 @@
     <link rel="stylesheet" href="../../css/pattern-scaffolding.css?{{ cacheBuster }}" media="all" />
     <link rel="stylesheet" href="https://db-ui.github.io/core/css/pattern-scaffolding-project-specific.css?{{ cacheBuster }}" media="all" />
 
-    <!-- Begin Pattern Lab (Required for Pattern Lab to run properly) -->
+	  <script type="module">
+		  import { defineCustomElements } from 'https://cdn.jsdelivr.net/npm/@db-ui/elements/loader/index.es2017.js';
+		  defineCustomElements();
+	  </script>
+	  <link rel="stylesheet"
+			href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.6.0/styles/default.min.css">
+	  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.6.0/highlight.min.js"></script>
```

Comment:
let's align this with jsdelivr that we're even already using:
[user-content-jsdelivr-link](https://www.npmjs.com/package/highlight.js#user-content-jsdelivr-link)

Reply:
fixed